imagersite/imagersite/static/styles/_base.scss

```diff
@@ -0,0 +1,40 @@
+
+p, a {
+  font-family: 'Montserrat';
+  font-size: 16px;
+  line-height: 1.1em;
+}
+
+a {
+  // color: #19532a;
+  text-decoration: none;
+}
+
+a:hover {
+  // color: #19532a;
+  text-decoration: underline;
+}
```

Comment:
You are not taking advantage of SCSS syntax. This can be written as
```scss
a {
  text-decoration: none;

  &:hover {
    text-decoration: underline;
  }
}
```